scripts/convert_hf_checkpoint.py

```diff
@@ -108,7 +108,7 @@ def copy_weights_falcon(
         state_dict[to_name] = param
 
 
-def copy_weights_open_llama(
+def copy_weights_open_LLaMA(
```

Comment:
```suggestion
def copy_weights_open_llama(
```
lowercase is ok here, it's mostly for class names